opensearch/opensearch.yml

```diff
@@ -0,0 +1,28 @@
+## Cluster Settings
+cluster.name: stac-cluster
+node.name: os01
+network.host: 0.0.0.0
+transport.host: 0.0.0.0
+discovery.type: single-node
+http.port: 9202
+http.cors.enabled: true
+http.cors.allow-headers: X-Requested-With,Content-Type,Content-Length,Accept,Authorization
+
+path:
+  repo:
+    - /usr/share/opensearch/snapshots
+    
+action.auto_create_index: true
+cluster.blocks.create_index: false
+
+cluster.routing.allocation.disk.watermark.low: 95%
+cluster.routing.allocation.disk.watermark.high: 97%
+cluster.routing.allocation.disk.watermark.flood_stage: 98%
+
```

Comment:
Can we use absolute values here instead? With a 2 TB disk, you'd go at 40GB into "flood stage". I think that 40GB disk space left isn't critical. But if you only have 10 GB disk space alltogheter then you'd go into "flood stage" at 200 MB. Then you are already in trouble.